app/models/user.rb

```diff
@@ -5,14 +5,15 @@ class User < ActiveRecord::Base
 
   PERMITTED_ATTRS = [:bio, :email, :handle, :hide_email, :location, :password, :website].freeze
 
+  before_destroy :yank_gems
```

Comment:
Previously, I was just using destroying the records from the database.  @qrush suggested that:
> Instead of destroying gems, the existing gems on an account should go through the Deletion process that yanked gems go through.

If the user being destroyed is the only owner of a gem then that gem will go through the Deletion process. ie it will be yanked.
I suppose it would also automate the process of namespace being taken over?